src/app/list/input-field/input-field.component.ts

```diff
@@ -0,0 +1,18 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+
+@Component({
+  selector: 'app-input-field',
+  templateUrl: './input-field.component.html',
+  styleUrls: ['./input-field.component.css']
+})
+export class InputFieldComponent {
+  inputValue = '';
+  @Output() newElement = new EventEmitter<string>();
+
+  addNewElement() {
+    if (this.inputValue.trim()) {
+      this.newElement.emit(this.inputValue.trim());
```

Comment:
probably this user input is not debounced => always debounce input fields 
```ts
this.Form.valueChanges.pipe(debounceTime(300),filter(),map(),...)
```

Reply:
I use a single input field in a standalone mode, so no form build. I will check how I can apply debounce in this case